CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.10.1
+
+A patch release, which fixes/improves some job metadata fields:
+
+- `dist_git_branch` (see `0.10.0`) has been renamed to `dist_git_branches` and it accepts also a list of values.
```

Comment:
Does it make sense to have a same naming for `branch`? (Rename it to `branches`?)

Reply:
Depends whether it can be only one value (branch) or list of values (branches).

Reply:
A single string for now but makes sense to support list as well.